lib/remote/consolehandler.cpp

```diff
@@ -5,23 +5,22 @@
 #include "remote/httputility.hpp"
 #include "remote/filterutility.hpp"
 #include "config/configcompiler.hpp"
-#include "base/configtype.hpp"
+#include "base/application.hpp"
 #include "base/configwriter.hpp"
 #include "base/scriptglobal.hpp"
 #include "base/logger.hpp"
 #include "base/serializer.hpp"
 #include "base/timer.hpp"
 #include "base/namespace.hpp"
-#include "base/initialize.hpp"
 #include "base/utility.hpp"
+#include <boost/smart_ptr/shared_ptr.hpp>
 #include <boost/thread/once.hpp>
-#include <set>
 
 using namespace icinga;
 
 REGISTER_URLHANDLER("/v1/console", ConsoleHandler);
 
-static std::map<String, ApiScriptFrame> l_ApiScriptFrames;
+static std::map<String, boost::shared_ptr<ApiScriptFrame>> l_ApiScriptFrames;
```

Comment:
Please prefer `std::shared_ptr` (unless there's something boost can do that the STL can't do).